.github/workflows/ci-macos.yml

```diff
@@ -33,22 +33,13 @@ jobs:
       with:
         python-version: '3.x'
 
-    - name: Run test script (without Homebrew)
+    - name: Run test script
       run: scripts/ci-github.sh
       shell: bash
       env:
         BRANCH: ${{ github.base_ref }}
         INDEX: ""
 
-    - name: Run test script (with Homebrew)
-      run: |
-        eval $(brew shellenv)
-        scripts/ci-github.sh
-      shell: bash
-      env:
-        BRANCH: ${{ github.base_ref }}
-        INDEX: ""
```

Comment:
So I understand that now tests run by default without either hombrew or macports, unless we purposely activate one of those in the tests themselves?

Reply:
I see, thanks for the explanation. I'm going to merge this one for now. I'm not sure how critical is to test the three situations; it might be overkill (for example we don't test Windows without msys2 IIRC).